### PR TITLE
first part of sending mails when shipping

### PR DESCRIPTION
**Pull request by ewelkaw**

### CODE REVIEW

File: ewardrobe/ewardrobe_app/admin.py [hunk 1]
```diff
@@ -86,6 +93,20 @@ def basket_actions(self, basket):
     def count_closing_date(date_modified):
         return (datetime.now().date() - date_modified).days > 14
 
+    @staticmethod
+    def send_confirmation_mail(name, email_adress, number):
```
Comment (esse) [inline ewardrobe/ewardrobe_app/admin.py]: `address` ;)

File: ewardrobe/ewardrobe_app/admin.py [hunk 2]
```diff
@@ -86,6 +93,20 @@ def basket_actions(self, basket):
...
+        email = EmailMessage(
+            f"{name} Your order has been shipped!",
+            f"""The order number {number} has ben already sent to you and should be delivered anytime now!
+            Remember that you have 14 days to return it.""",
+            settings.EMAIL_ADDRESS,
+            [email_adress],
+        )
+        log.warn(
```
Comment (esse) [inline ewardrobe/ewardrobe_app/admin.py]: why `warn`?

Comment (ewelkaw) [reply ewardrobe/ewardrobe_app/admin.py]: info

File: ewardrobe/ewardrobe_app/admin.py (hunk 1 above)
Comment (ewelkaw) [reply ewardrobe/ewardrobe_app/admin.py]: fixed

**Pull request merged**